source/install.txt

```diff
@@ -10,168 +10,406 @@ Install Compass
    :depth: 1
    :class: singlecol
 
-Compatibility
+Choose the tab corresponding to the platform on which you will install
+|compass-short| and follow the given tutorials.
+
+.. tabs::
+
+   tabs:
+     - id: osx
+       name: macOS
+       content: |
+
+     - id: windows
+       name: Windows
+       content: |
+
+     - id: ubuntu
+       name: Ubuntu
+       content: |
+
+
+     - id: rhel
+       name: RHEL
+       content: |
+
+
+Prerequisites
 -------------
 
-|compass| works with MongoDB Server version 3.0 and higher, both
-Enterprise and Community editions. It runs on 64-bit versions
-of Mac OS X, Windows, Ubuntu 14.04+ and RHEL 7+.
+Software Requirements
+~~~~~~~~~~~~~~~~~~~~~
+
+.. tabs::
+
+   hidden: true
+
+   tabs:
+     - id: osx
+       name: macOS
+       content: |
+
+         |compass-short| requires the 64-bit version of macOS 10.10 or
+         later and MongoDB 3.0 or later.
+
+     - id: windows
+       name: Windows
+       content: |
+
+         |compass-short| requires the 64-bit version of Microsoft
+         Windows 7 or later and MongoDB 3.0 or later.
+
+         .. note::
+
+            The Windows installation of |compass| requires **Microsoft
+            .NET Framework version 4.5 or later.** The |compass-short|
+            installer prompts you to install the minimum required
+            version of the .NET framework if it is not already installed
+            on your system.
+
+     - id: ubuntu
+       name: Ubuntu
+       content: |
+
+         |compass-short| requires the 64-bit version of Ubuntu 14.04 or
+         later and MongoDB 3.0 or later.
 
-Software requirements to run |compass-short| are listed below:
+     - id: rhel
+       name: RHEL
+       content: |
 
-.. include:: /includes/fact-compass-platforms.rst
+         |compass-short| requires the 64-bit version of RHEL 7+ or later
+         and MongoDB 3.0 or later.
 
-Installation
-------------
+Network Requirements
+~~~~~~~~~~~~~~~~~~~~
 
-.. note::
+Certain features of |compass|, such as third-party mapping services and
+Intercom, require inbound connections from external sources to
+function. In some instances, users have reported issues with their
+system firewall blocking these network requests and thereby limiting
+|compass| functionalities. To fix this issue, configure your system
+firewall to allow incoming connections for |compass|.
 
-   As of MongoDB Server version 3.6, MongoDB packages a
-   platform-specific |compass-short| installation script with
-   `MongoDB Server downloads
-   <https://www.mongodb.com/download-center/enterprise>`_. For details,
-   see the :manual:`install_compass script
-   </reference/program/install_compass/>` documentation.
+Ensure ports ``80`` and ``443`` are open between |compass| and the
```

Comment:
I might consider instead breaking up the paragraphs before "To fix this issue..." since the two sentences are so closely related it might make sense to have them in the same graf.

Reply:
@jeff-allen-mongo I'm confused, do you mean going from (1) to (2)? (described below):

**(1)**

> Certain features of MongoDB Compass, such as third-party mapping services and Intercom, require inbound connections from external sources to function. In some instances, users have reported issues with their system firewall blocking these network requests and thereby limiting MongoDB Compass functionalities. To fix this issue, configure your system firewall to allow incoming connections for MongoDB Compass.

> Ensure ports 80 and 443 are open between MongoDB Compass and the host for the MongoDB Instance to which Compass connects.


**(2)**

> Certain features of MongoDB Compass, such as third-party mapping services and Intercom, require inbound connections from external sources to function. In some instances, users have reported issues with their system firewall blocking these network requests and thereby limiting MongoDB Compass functionalities. 

> To fix this issue, configure your system firewall to allow incoming connections for MongoDB Compass. Ensure ports 80 and 443 are open between MongoDB Compass and the host for the MongoDB Instance to which Compass connects.

Reply:
Yes, precisely. Apologies for not being more explicit.

Reply:
Agreed. Fixed.